mce-io.c

```diff
@@ -370,11 +369,9 @@ static gboolean io_chunk_cb(GIOChannel *source,
 
 	chunk = g_malloc(iomon->chunk_size);
 
-	do {
```

Comment:
why is that loop removed?

Reply:
this loop can cause a deadlock. Evdev uses EAGAIN to signify that no event is currently available, this state persists until the next event is triggerd on the event device, if for any reason the mce code becomes unaligned (in respect to sizeof(event) blocks with the event file (such as when the input layer is not done writing to the file), mce will get stuck in an endless loop until the user presses another key.